### PR TITLE
Add missing AppStatus enum values to api-docs.yaml

### PR DESCRIPTION
**Pull request by jmthibault79**
These are present in kubernetesModel.scala

---
Have you read [CONTRIBUTING.md](https://github.com/DataBiosphere/leonardo/blob/develop/CONTRIBUTING.md) lately? If not, do that first.

I, the developer opening this PR, do solemnly pinky swear that:

- [ ] I've documented my API changes in Swagger

In all cases:

- [ ] Get a thumbsworth of review and PO signoff if necessary
- [ ] Verify all tests go green (you can re-run automation tests with a comment saying `jenkins retest`
- [ ] Run the automation tests multiple times in parallel to weed out instability if applicable via a comment saying `multi-test`
- [ ] Squash and merge; Delete your branch after this
- [ ] Test this change deployed correctly and works on dev environment after deployment

### CODE REVIEW

File: http/src/main/resources/swagger/api-docs.yaml [hunk 1]
```diff
@@ -2625,11 +2625,16 @@ components:
     AppStatus:
       type: string
       enum:
-        - DELETING
         - DELETED
+        - DELETING
         - ERROR
+        - PRECREATTING
```
Comment (LizBaldo) [inline http/src/main/resources/swagger/api-docs.yaml]: Nit, `PRECREATING`

**Review by LizBaldo — APPROVED** [review]
Thanks for adding these!

File: http/src/main/resources/swagger/api-docs.yaml (hunk 1 above)
Comment (Qi77Qi) [reply http/src/main/resources/swagger/api-docs.yaml]: oh, we really shouldn't be exposing these statues..I'll have a separate PR to not exposing these details to users

Comment (jmthibault79) [reply http/src/main/resources/swagger/api-docs.yaml]: thanks.  Should I fix this spelling or remove all of the PRE ?

Comment (jmthibault79) [reply http/src/main/resources/swagger/api-docs.yaml]: I removed the PRE.  Now it's just adding STARTING and alphabetizing.

**Review by Qi77Qi — APPROVED** [review]
thanks!

**Pull request merged**